src/index.html

```diff
@@ -8,6 +8,90 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars stars--0">
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+    </div>
+
+    <div class="stars stars--1">
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+    </div>
+
+    <div class="stars stars--2">
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+    </div>
+
+    <div class="stars stars--3">
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+    </div>
+
+    <div class="stars stars--4">
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+    </div>
+
+    <div class="stars stars--5">
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+    </div>
+
```

Comment:
```suggestion
    <div class="stars stars--5">
      <div class="stars__star"></div>

      <div class="stars__star"></div>

      <div class="stars__star"></div>

      <div class="stars__star"></div>

      <div class="stars__star"></div>
    </div>
```
for better readability add the blank line between sibling tags but not between parent and child tags